src/Humanizer/Properties/Resources.resx

```diff
@@ -579,4 +579,100 @@
   <data name="TimeSpanHumanize_MultipleDays_DualTrialQuadral" xml:space="preserve">
     <value>{0} days</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="N" xml:space="preserve">
+    <value>north</value>
+  </data>
+  <data name="NNE" xml:space="preserve">
+    <value>north-northeast</value>
+  </data>
+  <data name="NE" xml:space="preserve">
+    <value>northeast</value>
+  </data>
+  <data name="ENE" xml:space="preserve">
+    <value>east-northeast</value>
+  </data>
+  <data name="E" xml:space="preserve">
+    <value>east</value>
+  </data>
+  <data name="ESE" xml:space="preserve">
+    <value>east-southeast</value>
+  </data>
+  <data name="SE" xml:space="preserve">
+    <value>southeast</value>
+  </data>
+  <data name="SSE" xml:space="preserve">
+    <value>south-southeast</value>
+  </data>
+  <data name="S" xml:space="preserve">
+    <value>south</value>
+  </data>
+  <data name="SSW" xml:space="preserve">
+    <value>south-southwest</value>
+  </data>
+  <data name="SW" xml:space="preserve">
+    <value>southwest</value>
+  </data>
+  <data name="WSW" xml:space="preserve">
+    <value>west-southwest</value>
+  </data>
+  <data name="W" xml:space="preserve">
+    <value>west</value>
+  </data>
+  <data name="WNW" xml:space="preserve">
+    <value>west-northwest</value>
+  </data>
+  <data name="NW" xml:space="preserve">
+    <value>northwest</value>
+  </data>
+  <data name="NNW" xml:space="preserve">
+    <value>north-northwest</value>
+  </data>
+  <data name="N_Short" xml:space="preserve">
+    <value>n</value>
```

Comment:
Should short abbreviations be caps?

Reply:
Because all other string values are lowercase I chose to do these in lowercase, too. Not sure what would be preferred in daily use cases though.